com.woltlab.wcf/templates/footer.tpl

```diff
@@ -43,7 +43,13 @@
 			{/capture}
 				
 			{if $__sidebarRightContent|trim}
-				<aside class="sidebar boxesSidebarRight" data-show-sidebar="{lang}wcf.global.button.showSidebar{/lang}" data-hide-sidebar="{lang}wcf.global.button.hideSidebar{/lang}">
+				{if !$sidebarRightShow|isset}
```

Comment:
Rename this variable to `$__sidebarRightShow` in order to avoid conflicts with existing variables. This applies to `$sidebarRightHide` (new: `$__sidebarRightHide`) as well as the variables in the `header.tpl` too.

Additionally, turn these `if`-statements into one-liners, splitting them across three lines doesn't really improve readability.